src/equations/compressible_euler_3d.jl

```diff
@@ -765,6 +765,8 @@ end
 
 
 # Rotate normal vector to x-axis; normal, tangent1 and tangent2 need to be orthonormal
+# Called inside `FluxRotated` in `numerical_fluxes.jl` so the directions
+# has been normalized prior to this rotation of the state vector
 @inline function rotate_to_x(u, normal, tangent1, tangent2, equations::CompressibleEulerEquations3D)
```

Comment:
Is this `normal_vector` or `normal_direction`?

It would also be good to add a small entry to the docs (maybe [here](https://trixi-framework.github.io/Trixi.jl/stable/conventions/#Spatial-dimensions-and-directions)) that says explains when we use `normal_vector` and when `normal_direction` - what do you think?

Reply:
This is the normal vector. I will also update the docs in the suggested part

Reply:
LGTM, thanks!